src/index.js

```diff
@@ -92,6 +105,59 @@ function determineTrip(event) {
     return +event.target.id.split('-')[1] === trip.id
   })
   if (trip) {
-    domUpdates.displayTripDetails(trip)
+    domUpdates.displayTripDetails(trip, false)
   }
 }
+function getDestinationsList() {
+  let destinationList = destinationsData.reduce((list, destination) => {
+    if (!list.includes(destination.destination)) {
+      list.push(destination.destination)
+    }
+    return list
+  }, [])
+  return destinationList.sort()
+}
+function displayBookingView() {
+  let destinationList = getDestinationsList()
+  domUpdates.changePageDisplay('planTripView')
+  domUpdates.displayDestinationDropdown(destinationList)
+}
+function displayNewTrip(event) {
+  event.preventDefault()
+  const travelerCount = +document.querySelector('#travelersAmount').value
+  const startDate = document.querySelector('#daySelector').value
+  const duration = +document.querySelector('#tripDuration').value
+  const destination = document.querySelector('#destinationInput').value
+  let destinationData = findDestination(destination)
+  buildNewTrip(travelerCount, startDate, duration, destination)
+  let trip = new Trip(newTrip, destinationData)
+  domUpdates.displayTripDetails(trip, true)
+}
+function buildNewTrip(travelerCount, startDate, duration, destination) {
+  newTrip.id = Date.now()
+  newTrip.userID = currentTraveler.id
+  newTrip.destinationID = findDestination(destination).id
+  newTrip.travelers = travelerCount
+  newTrip.date = moment(new Date(startDate)).format("YYYY/MM/DD")
+  newTrip.duration = duration
+  newTrip.status = 'pending'
+  newTrip.suggestedActivities = []
+}
+function findDestination(destination) {
+  if (typeof destination === 'string') {
```

Comment:
Is destination ever not a string? If this is a null check you can just put `if (destination)`